patches/websdk/0001-Add-versioned-AspNetCore-patch-override-properties.patch

```diff
@@ -0,0 +1,45 @@
+From 2e31a43df045b006135fd9be3e602ebeb65c49a2 Mon Sep 17 00:00:00 2001
+From: Davis Goodin <dagood@microsoft.com>
+Date: Wed, 13 Jun 2018 09:53:41 -0500
+Subject: [PATCH] Add versioned AspNetCore patch override properties
+
+https://github.com/aspnet/websdk/issues/355 will include changes that make this patch unnecessary.
+---
+ .../Sdk.DefaultItems.targets                         | 12 ++++--------
+ 1 file changed, 4 insertions(+), 8 deletions(-)
+
+diff --git a/src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.DefaultItems.targets b/src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.DefaultItems.targets
+index 9fc331e..3e6239b 100644
+--- a/src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.DefaultItems.targets
++++ b/src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.DefaultItems.targets
+@@ -67,11 +67,9 @@ Copyright (c) .NET Foundation. All rights reserved.
+       <!-- Latest patch version of .All Framework -->
+       <PropertyGroup Condition="'$(LatestAspNetCoreAllPatchVersion)' == ''">
+         <!-- Placeholder for setting latest patch version using the bundled version from CLI for previous TFMs, this will not apply until 2.2 -->
+-        <!-- <LatestAspNetCoreAllPatchVersion Condition="'$(_TargetFrameworkVersionWithoutV)' == '2.1'">$(LatestPatchVersionForAspNetCoreAll2_1)</LatestAspNetCoreAllPatchVersion> -->
++        <LatestAspNetCoreAllPatchVersion Condition="'$(_TargetFrameworkVersionWithoutV)' == '2.1'">$(LatestPatchVersionForAspNetCoreAll2_1)</LatestAspNetCoreAllPatchVersion>
++        <LatestAspNetCoreAllPatchVersion Condition="'$(_TargetFrameworkVersionWithoutV)' == '2.2'">$(LatestPatchVersionForAspNetCoreAll2_2)</LatestAspNetCoreAllPatchVersion>
```

Comment:
This patch isn't fully working yet (see https://github.com/dotnet/source-build/issues/586#issuecomment-397011217) so I'm going to go ahead and merge https://github.com/dotnet/source-build/pull/598 since its CI finished and rebase this PR with the fix.